src/display/display_sdl.c

```diff
@@ -600,6 +600,29 @@ static int has_unicode_event_queued()
 	return false;
 }
 
+/* Determine if there is an event waiting in the queue, and if so,
+   delete it. This is used (with SDL_TEXTINPUT) to keep the literal
+   part of a hotkey from being passed through when dealing with
+   hotkeys. (E.g. we don't want Alt+S to also return 's'.)
+   It's also used with SDL_KEYDOWN to remove stray TAB presses due
+   to ALT+TAB not being properly handled on some Linux wms. */
+static void clear_event(SDL_EventType event_type)
+{
+	const int MAX_EVENTS = 10;
+	SDL_Event outevent[MAX_EVENTS];
+	int pending_events = SDL_PeepEvents(outevent,
+		MAX_EVENTS, SDL_PEEKEVENT, event_type,
+		event_type);
+
+	if (pending_events == 0)
+		return;
+
+	SDL_PeepEvents(outevent, 1, SDL_GETEVENT,
+		event_type, event_type);
+
+	return;
```

Comment:
Since we're only looking for one type of event (minType == maxType == event_type) and then immediately getting it (and discarding it), we can just use `SDL_GETEVENT` and ignore the return value:

```suggestion
	SDL_Event outevent;
	SDL_PeepEvents(&outevent, 1, SDL_GETEVENT,
		event_type, event_type);
```